spec/services/session_encryptor_spec.rb

```diff
@@ -3,15 +3,27 @@
 describe SessionEncryptor do
   describe '#load' do
     it 'decrypts encrypted session' do
-      session = SessionEncryptor.dump(foo: 'bar')
+      session = SessionEncryptor.new.dump(foo: 'bar')
 
-      expect(SessionEncryptor.load(session)).to eq('foo' => 'bar')
+      expect(SessionEncryptor.new.load(session)).to eq('foo' => 'bar')
     end
   end
 
+  it 'makes a round trip okay' do
```

Comment:
we may want to say "makes a roundtrip okay across separate instances" but this is fine